src/developer-documentation/ibc/ibc.md

```diff
@@ -0,0 +1,237 @@
+
+# 1 IBC infomation
+## 1.1 Channel lifecycle management
+
+![](../../.gitbook/assets/ibc/CSM.png)
+
+
+# 2 ICS20 interface
+## 2.1 ink! interface
+```js
+        /// transfer token, This allows us to transfer *exactly one* native token
+        pub fn execute_transfer(
+            &self,
+            msg: TransferMsg,
+            amount: Amount,
+            sender: Addr,
+        ) -> Result<Response, Error> ;
+
+
+        /// Return the port ID bound by this contract.
+        pub fn query_port(&self) -> PortResponse ;
+```
```

Comment:
```suggestion
```rust
/// Transfer token; this allows us to transfer *exactly one* native token.
pub fn execute_transfer(
    &self,
    msg: TransferMsg,
    amount: Amount,
    sender: Addr,
) -> Result<Response, Error>;

/// Return the port ID bound by this contract.
pub fn query_port(&self) -> PortResponse;
```
```

Reply:
sure